internal/ingress/controller/util.go

```diff
@@ -95,3 +98,58 @@ func compareRoute(r1, r2 *kong.Route) bool {
 
 	return true
 }
+
+func mergeRouteAndKongIngress(r *kong.Route, kongIngress *configurationv1.KongIngress) bool {
+	if r == nil {
+		return false
```

Comment:
should this be true?

Reply:
The return value here indicates if the the Route passed into this method was modified or not, which in this case, it's not.

I might be missing something here.